build.gradle

```diff
@@ -114,7 +133,11 @@ runIde {
 }
 
 runIdeForUiTests {
+    systemProperty "robot-server.port", System.getProperty("robot-server.port", "8580")
     systemProperties['com.redhat.devtools.intellij.telemetry.mode'] = 'debug'
+    systemProperty "idea.trust.all.projects", "true"
```

Comment:
please re-order new properties below existing and use same layout, without the parenthesis.